dbt/dbt_project.yml

```diff
@@ -52,8 +52,6 @@ models:
     +materialized: view
     +write_compression: zstd
     +format: parquet
-    ccao:
-      +schema: ccao
```

Comment:
As with the `rpie` schema, if we leave it in, we get a warning that no models are configured to use the schema:

```
16:26:41  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.ccao_data_athena.ccao
```

It doesn't seem to affect dbt's behavior either way, but I figured we might as well silence the warning.

Reply:
As long as it doesn't affect the creation of the `ccao` seeds!

Reply:
Seeds should be unaffected since they're configured under a separate `seeds` attribute:

https://github.com/ccao-data/data-architecture/blob/aebf2752056da89b9d47bd338450739d80ce47b8/dbt/dbt_project.yml#L78-L81